src/sentry/options/defaults.py

```diff
@@ -1455,3 +1455,10 @@
     default=[],
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+
+# flag for turning on/off the options_automator_slack_webhook.
+register(
+    "options_automator_slack_webhook",
+    default=False,
```

Comment:
Please set default = `True`.
This is meant to disable in case of emergency. So the default should be on.